Makefile

```diff
@@ -117,7 +117,7 @@ sync-image: sync $(IMAGEBUILDER) pullregistry
 
 .PHONY: sync-run
 sync-run: generate
-	go run -ldflags ${LDFLAGS} ./cmd/sync -run-once
+	go run -ldflags ${LDFLAGS} ./cmd/sync -run-once -loglevel Debug
```

Comment:
under normal operations the sync pod container runs on the default Info level, manually running the sync pod runs with Debug level